### ISSUE TITLE
Non-Sanity test case test_attach_detach_with_corrupt_stateless_mme.py fails in s1-setup

### ISSUE THREAD

**Issue by VinashakAnkitAman** [magma/magma#5991]
### Your Environment

- **Version:** Local Development Setup
- **Affected Component:** Access Gateway
- **Affected Subcomponent:** Magma MME/SPGW service
- **Deployment Environment:** Vagrant (AGW)

### Setup Details
The S1APTester ([https://github.com/facebookexperimental/S1APTester](https://github.com/facebookexperimental/S1APTester)) was used along with Magma EPC in Vagrant based setup to reproduce the issue.

### Describe the Issue
This issue is a subset of issue #5842 (Also refer PR #5840 which adds all these non-sanity test cases as part of defs.mk file)

The non-sanity test cases **s1aptests/test_attach_detach_with_corrupt_stateless_mme.py** is failing while doing the S1-Setup. Following is the error log in the terminal

```
************************* S1 setup
Deleting Ue Context from Traffic Handler
************************* send SCTP SHUTDOWN
Keys left in Redis (list should be empty)[
 IMSI001010000000001:SPGW
IMSI001010000000001.magma.ipv4,ipv4:mobilityd_ipdesc_record
IMSI001010000000001:MME

nb_enb_connected: 1
nb_ue_attached: 1
]
Entries left in hashtables (should be zero): 5
F
======================================================================
FAIL: Basic attach/detach test with two UEs,
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vagrant/magma/lte/gateway/python/integ_tests/s1aptests/test_attach_detach_with_corrupt_stateless_mme.py", line 71, in test_attach_detach_with_corrupt_stateless_mme
    self._s1ap_wrapper._s1setup()
  File "/home/vagrant/magma/lte/gateway/python/integ_tests/s1aptests/s1ap_wrapper.py", line 148, in _s1setup
    res = self._issue_s1setup_req()
  File "/home/vagrant/magma/lte/gateway/python/integ_tests/s1aptests/s1ap_wrapper.py", line 142, in _issue_s1setup_req
    assert response.msg_type == s1ap_types.tfwCmd.ENB_S1_SETUP_RESP.value
AssertionError:
-------------------- >> begin captured logging << --------------------
```

After looking into the S1APTester log, it can be seen that ENB is failing to send S1-Setup message with following error:
```
[2021_4_7_14_12_39.033][FW]../src/tfwApp/fw_api_int.c:1821      DEBUG:  Entering tfwApi()
[2021_4_7_14_12_39.033][FW]../src/tfwApp/fw_api_int.c:1864      INFO:  S1 SETUP-REQ
[2021_4_7_14_12_39.033][FW]../src/tfwApp/fw_api_int.c:1875      ERROR:  FAILED TO SEND S1 SETUP REQ: ENAPP IS ALREADY UP
[2021_4_7_14_12_39.033][FW]../src/tfwApp/fw_api_int.c:2442      DEBUG:  Exiting tfwApi(), [Return 0]
[2021_4_7_14_14_57.991][NB]../src/enbApp/nb_dam_tmr.c:330       ERROR:  Inactivity Timer Expired for UE 1: Current Expiry Count = 1 MAX_EXPIRY_COUNT = 3
[2021_4_7_14_16_57.991][NB]../src/enbApp/nb_dam_tmr.c:330       ERROR:  Inactivity Timer Expired for UE 1: Current Expiry Count = 2 MAX_EXPIRY_COUNT = 3
```

The ENB state in S1APTester is already in S1-Setup mode, hence it is discarding any new S1-Setup connection. The pcap is attached here along with S1APTester logs which shows there are no other communication to ENB from MME side also to change the ENB S1-State.
[S1APTester.zip](https://github.com/magma/magma/files/6270892/S1APTester.zip)

Referring the previous PRs with changes on this test script #4015 and #5264. 

**To Reproduce**
1) Run the test case individually "make integ_test TESTS=<testcase_name>"

**Comment by VinashakAnkitAman** [comment]
@ardzoht I see that this test case was working fine as part of your PR #4015 as per the terminal log you have attached. But for me I don't see any reason for the test case to pass in current test script and code as the ENB state does not change in between for the next S1-Setup to be allowed. 

Do you have any packet capture regarding this test script or if you can help me understand was there step in between (like SCTP shutdown/abort) which could change the S1-state in ENB (S1APTester) for the new S1-Setup to pass?

**Comment by ardzoht** [comment]
Hi @VinashakAnkitAman, there was an error on how the health service was being enabled and disabled on `s1ap_utils`, this is why the s1ap test was failing, this was fixed on #6247, can you confirm that the test passes for you? If it does we can close this issue, thanks!

**Comment by VinashakAnkitAman** [comment]
Thank you @ardzoht for the update. I have verified the test case by running it multiple times in loop and it seems to work fine for me, hence closing the issue.

Attaching the successful terminal log for reference: 
[success_log_corrupt_state_test_case.txt](https://github.com/magma/magma/files/6353563/success_log_corrupt_state_test_case.txt)
